packages/webamp-modern-2/src/UIRoot.ts

```diff
@@ -170,8 +171,12 @@ export class UIRoot {
       }
       const url = imgCache.get(groupId);
       // TODO: Techincally we only need one per image/gammagroup.
-      this._div.style.setProperty(bitmap.getCSSVar(), `url(${url})`);
+      cssRules.push(`  ${bitmap.getCSSVar()}: url(${url});`)
     }
+    cssRules.unshift(':root{')
+    cssRules.push('}')
+    const cssEl = document.getElementById('bitmap-css');
+    cssEl.textContent = cssRules.join("\n");
```

Comment:
Maybe a little cleaner, but not blocking.

```suggestion
    const cssEl = document.getElementById('bitmap-css');
    cssEl.textContent = `:root { ${cssRules.join("\n")} }`;
```

Reply:
I did it and doesn't work!
![image](https://user-images.githubusercontent.com/1678529/158952790-c26963b9-e559-41fb-8b2d-924946d95d2a.png)

Reply:
Maybe it needs the newlines?

Reply:
It's fine to leave it as is.

Reply:
Like any other property, this is written inside a ruleset, like so:

```css
element {
  --main-bg-color: brown;
}```
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#basic_usage